pkg/maps/nat/fake/per_cluster.go

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package nat
```

Comment:
Guessing this is used for testing? If so might make sense to only lasso this in while in test mode and thus use package nat_test instead.

Reply:
Yes, this is the fake implementation provided for testing purposes. AFAIU it is not possible to import the methods defined inside a `something_test.go` file from another package, so that would not work. Still, this PR already moves the fake implementation to a separate package (under the `fake` directory), so that it can be imported separately from the one with the real implementation.